pkg/patch/patch.go

```diff
@@ -847,7 +867,26 @@ func patchMultiPlatformImage(
 		}
 	}
 
-	log.Infof("Multi-platform image patched with tag %s", patchedImageName.String())
+	var b strings.Builder
+	w := tabwriter.NewWriter(&b, 0, 0, 2, ' ', 0)
+	fmt.Fprintln(w, "PLATFORM\tSTATUS\tREFERENCE\tERROR")
+
+	for _, p := range platforms {
+		platformKey := p.OS + "/" + p.Architecture
```

Comment:
it looks like this is duplicated a few times, maybe we can pull it out into a shared function? There is an existing key in https://github.com/project-copacetic/copacetic/blob/main/pkg/buildkit/buildkit.go#L232. I like the conditional check for Variant. maybe we can combine these and place the function in buildkit.go and then import it here?